controllers/osbuild_controller.go

```diff
@@ -18,19 +20,51 @@ package controllers
 
 import (
 	"context"
+	"fmt"
+	"github.com/go-logr/logr"
+	"net/http"
+	"time"
 
+	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/log"
 
-	osbuilderprojectflottaiov1alpha1 "github.com/project-flotta/osbuild-operator/api/v1alpha1"
+	osbuildv1alpha1 "github.com/project-flotta/osbuild-operator/api/v1alpha1"
+	"github.com/project-flotta/osbuild-operator/internal/composer"
+	repositoryosbuild "github.com/project-flotta/osbuild-operator/internal/repository/osbuild"
+)
+
+const (
+	// Conditions Messages
+	FailedToSendPostRequestMsg      = "Failed to post a new composer build request"
+	EdgeContainerJobFinishedMsg     = "Edge-container job was finished successfully"
+	EdgeContainerJobFailedMsg       = "Edge-container job was failed"
+	EdgeContainerJobStillRunningMsg = "Edge-container job was still running"
+
+	// OSBuildConditionTypes values
+	ContainerBuildDone    = "containerBuildDone"
+	FailedContainerBuild  = "failedContainerBuild"
+	StartedContainerBuild = "startedContainerBuild"
+	IsoBuildDone          = "isoBuildDone"
+	FailedIsoBuild        = "failedIsoBuild"
+	StartedIsoBuild       = "startedIsoBuild"
+
+	// Image types
+	EdgeContainerImgType = "edge-container"
+	EdgeInstallerImgType = "edge-installer"
+
+	EmptyComposeID = ""
+
+	RequeueForTwoMinutes = time.Minute * 2
```

Comment:
Not sure you want to name the field bases on its value. What happens if we decide to change it to three minutes?

Reply:
On  a different PR I got the opposite approach - that someone preferred to mentioned the specific time